### PR TITLE
Changed ALL Names of objects, constants, etc... to Java naming convtion

### PR DESCRIPTION
**Pull request by KSatsujin**

### CODE REVIEW

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -18,33 +18,32 @@
...
-    public static final int kRightRearMotor = 4;
+    public static final int FRONT_LEFT_MOTOR = 1;
+    public static final int FRONT_RIGHT_MOTOR = 2;
+    public static final int REAR_LEFT_MOTOR = 3;
+    public static final int REAR_RIGHT_MOTOR = 4;
 
     // Speed Calibration
-    public static final double kSlowmodeSpeed = 0.5;
-    public static final double kDefaultMaxAcceleration = 5.0;
+    public static final double SLOWMODE_SPEED = 0.5;
```
Comment (shanealv) [inline src/main/java/frc/robot/Constants.java]: Rename this to "DEFAULT_SLOW_MODE_SPEED"

**Review by shanealv — CHANGES_REQUESTED** [review]
One suggested change, otherwise things are fine.

File: src/main/java/frc/robot/Constants.java (hunk 1 above)
Comment (KSatsujin) [reply src/main/java/frc/robot/Constants.java]: Done

**Review by shanealv — APPROVED** [review]
Ready to merge as your leisure.

**Pull request merged**